main.go

```diff
@@ -0,0 +1,422 @@
+	TopicArn         string `json:"topicArn"`
+	Subject          string `json:"subject"`
+	Message          string `json:"message"`
+	Timestamp        string `json:"timestamp"`
+	SignatureVersion string `json:"signatureVersion"`
+	Signature        string `json:"signature"`
+	SigningCertURL   string `json:"signingCertURL"`
+	UnsubscribeURL   string `json:"unsubscribeURL"`
+}
+
+// Message is used to decode the SQS Message
+type Message struct {
+	AlarmName        string  `json:"alarmName"`
+	AlarmDescription string  `json:"alarmDescription"`
+	AWSAccountID     string  `json:"awsAccountId"`
+	NewStateValue    string  `json:"newStateValue"`
+	NewStateReason   string  `json:"newStateReason"`
+	StateChangeTime  string  `json:"stateChangeTime"`
+	Region           string  `json:"region"`
+	AlarmArn         string  `json:"alarmArn"`
+	OldStateValue    string  `json:"oldStateValue"`
+	Trigger          Trigger `json:"trigger"`
+}
+
+// Trigger is used to decode the Trigger component of the SQS Message
+type Trigger struct {
+	MetricName                       string       `json:"metricName"`
+	Namespace                        string       `json:"namespace"`
+	StatisticType                    string       `json:"statisticType"`
+	Statistic                        string       `json:"statistic"`
+	Unit                             string       `json:"unit"`
+	Dimensions                       []Dimensions `json:"dimensions"`
+	Period                           int          `json:"period"`
+	EvaluationPeriods                int          `json:"evaluationPeriods"`
+	ComparisonOperator               string       `json:"comparisonOperator"`
+	Threshold                        float32      `json:"threshold"`
+	TreatMissingData                 string       `json:"treatMissingData"`
+	EvaluateLowSampleCountPercentile string       `json:"evaluateLowSampleCountPercentile"`
+}
+
+// Dimensions is used to decode the Dimensions component of the Trigger
+type Dimensions struct {
+	Value string `json:"value"`
+	Name  string `json:"name"`
+}
+
+// DBInstance is used to store information about each DB Instance
+type DBInstance struct {
+	SizeIndex            int    `json:"sizeIndex"`
+	DBInstanceClass      string `json:"dbInstanceType"`
+	DBInstanceStatus     string `json:"dbInstanceStatus"`
+	DBInstanceIdentifier string `json:"dbInstanceIdentifier"`
+	DBClusterIdentifier  string `json:"dbClusterIdentifier"`
+	IsClusterWriter      bool   `json:"isClusterWriter"`
+}
+
+func main() {
+	err := verticalScaling()
+	if err != nil {
+		log.WithError(err).Error("Failed to run database factory vertical scaling")
+		err = sendMattermostErrorNotification(err, "Τhe Database Factory vertical scaling failed")
+		if err != nil {
+			log.WithError(err).Error("Failed to send Mattermost error notification")
+		}
+	}
+}
+
+func verticalScaling() error {
+	SQSClient, RDSClient, err := getAWSClients()
+	if err != nil {
+		return errors.Wrapf(err, "Failed to initiate AWS Clients")
```

Comment:
```suggestion
		return errors.Wrap(err, "failed to initiate AWS Clients")
```

Reply:
Many more like this below using `Wrapf`.

Reply:
Nice catch, thanks. Fixed everywhere :)